src/coldcard.ts

```diff
@@ -275,11 +280,13 @@ class ColdcardMultisigSettingsFileParser extends ColdcardInteraction {
    */
   deriveDeeperXpubIfNecessary(result) {
     const knownColdcardChroot = this.chrootForBIP32Path(this.bip32Path);
-    let relativePath = getRelativeBIP32Path(
-      knownColdcardChroot,
-      this.bip32Path
-    );
-    let addressType = COLDCARD_BASE_BIP32_PATHS[knownColdcardChroot];
+    let relativePath =
+      knownColdcardChroot &&
+      getRelativeBIP32Path(knownColdcardChroot, this.bip32Path);
+    let addressType;
+    if (knownColdcardChroot !== null) {
+      addressType = COLDCARD_BASE_BIP32_PATHS[knownColdcardChroot];
+    }
     // result could have p2wsh_p2sh or p2sh_p2wsh based on firmware version. Blah!
     if (addressType.includes("_") && !result[addressType.toLowerCase()]) {
```

Comment:
I like the additional checking above, but this line will throw if `addressType` is undefined. Give it an initializer on its declaration.